server/routerlicious/packages/services-core/src/database.ts

```diff
@@ -153,7 +153,7 @@ export interface IDb {
      * Removes a collection or view from the database.
      * The method also removes any indexes associated with the dropped collection.
      */
-    dropCollection(name: string): Promise<boolean>;
+    dropCollection?(name: string): Promise<boolean>;
```

Comment:
We can also add a new interface, say `IDropableDb` had put this method in.

Then in the databases that implement the `ID`, do multiple implementation like `export class MongoDb implements core.IDb, core.IDropableDb`.

How is that sound?

Reply:
Yup, that is also a viable solution.